### ISSUE TITLE
gcloud docker permission denied error on 17.06.0

### ISSUE THREAD

**Issue by nico-francois-vex** [docker/for-win#866]
### Expected behavior
`gcloud docker -- --authorize-only` or any other command should authorize docker for Google Cloud and push or pull as instructed.

### Actual behavior
There's a permission denied error when trying to run `gcloud docker`.

```$ gcloud docker -- --authorize-only
ERROR: gcloud crashed (IOError): [Errno 13] Permission denied: 'C:\\Users\\username\\.docker\\config.json'

If you would like to report this issue, please run the following command:
  gcloud feedback

To check gcloud for common problems, please run the following command:
  gcloud info --run-diagnostics
```

### Information

Diagnostic ID: DA399C3F-F804-4626-8630-FEC0BD1D4911/2017-06-29_16-29-25
Windows 10 Pro x64 15063.413

Issue requested by @dgageot

**Comment by dustinmoris** [comment]
I have the same issue and am not able to resolve it.

**Comment by mpskovvang** [comment]
I'm also having this issue after upgrading to 17.06. The changes to config.json by 'docker-credential-gcr configure-docker' is also reverting periodically.

Where can I find a download link for 17.03?

**Comment by edoniti** [comment]
I also have the same issue.

**Comment by magical-rogue** [comment]
I also have this issue

**Comment by charlesgetup** [comment]
I have same issue, too. Hope there is a fix for this.

**Comment by magical-rogue** [comment]
I managed to work around this by changing the permissions for the docker folder "CACLS .docker /e /p $user:f", I then got an err stating that the config.json was in use by another process, so after a bit of googling I found [someone had worked around that](https://github.com/docker/for-win/issues/867) by deleting the config.json. Once I had deleted the config.json I was able to gcloud docker push/pull etc

**Comment by mrts** [comment]
As @magical-rogue said, the fix is to `rm ~/.docker/config.json`.

**Comment by friism** [comment]
I'm pretty sure this is a dupe of https://github.com/docker/for-win/issues/867 closing here.

**Comment by friism** [comment]
We've shipped fixes for this in both Stable (17.06.1) and Edge (17.07). Please comment or re-open if you're still seeing the problem after updating.
